pages/linux/pacman.md

```diff
@@ -14,9 +14,9 @@
 
 `pacman -Rs {{package_name}}`
 
-- Search the package database for a keyword:
+- Search the package database for a regular expression or keyword:
 
-`pacman -Ss icon theme`
+`pacman -Ss {{regexp}}`
```

Comment:
Maybe wrap the token in quote marks? or it isn't needed? Also, perhaps {{search_pattern}} would be better, since regular expressions are already mentioned in the description.